pipelines/template/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""
+This pipeline provides data extraction from an example source as a starting point for new pipelines.
+"""
+
+import typing as t
+from typing import Sequence, Iterable, Dict, Any
+import dlt
+from dlt.common.typing import TDataItem
+from dlt.extract.source import DltResource
+
+FRUITS = [
```

Comment:
I'd do that but using Pokemon API :>

Reply:
I have used the chess pipeline in the contributing.md now